### PR TITLE
Implemented Test for Facial Recognition

### LINKED ISSUE #9: Implement Incomplete Test for Facial Recognition
### Description
The `test_facial_recognition` test case in the `test_security_system.py` file is currently incomplete and needs to be implemented. This task focuses on completing the test to ensure that facial recognition functionality is thoroughly tested.

### Expected behavior

1. **Facial Recognition Test (in `test_facial_recognition`):**
   - Implement the `test_facial_recognition` test case.
   - Ensure that the test case correctly checks the functionality of the `recognize_face` method.
   - Verify that recognized face labels are present in the processed image.
   - Provide thorough testing to validate facial recognition accuracy.

### Additional context
- The test case is part of the `test_security_system.py` file.
- The `setUp` method initializes the `SecuritySystem` object with a test model and cascade classifier.
- The existing test cases in the file check access logging and face detection functionality.

Completing this test will contribute to a comprehensive assessment of the facial recognition feature and help ensure its accuracy and reliability in the project.

### PR DESCRIPTION
**Pull request by Aqua-16**
## Description

I have fixed file path issues persisting in security_system.py file. The testing has been done through picking multiple images from a test folder, checking their true name and authorization status, and then comparing that with the values computed by the `recognize_face` method. This testing has been done in a way that more images can be freely added to the test folder without causing any issues to the code, as long as the same file naming convention is followed as the other images, which is

trueNameOfPerson_serialNumberOfImage.png

## Related Issue
Also somewhat fixes the tester for the logging function.

Fixes #9 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots (if appropriate):

image
The third '.' in '.E.' signifies that there are no errors in recognize_face.

## Checklist:

- [x] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
- [x] My code follows the code style of this project.
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/test_images/test_face.png
Comment (sanskaromar) [inline tests/test_images/test_face.png:1]: Upload minified jpg images.

**Review by sanskaromar — CHANGES_REQUESTED** [review]
As discussed, We need to make few changes

File: tests/test_images/test_face.png
Comment (Aqua-16) [reply tests/test_images/test_face.png:1]: Done, also added more images

**Comment by Aqua-16** [comment]
I have made it check multiple image files and also coded it to generally pull from the image directory instead of hard coding the path, so more images can be added. 

Also, I've not specifically mentioned a model to be used, so testing can be done on any model added to the specified directory.

File: app/security_system.py [hunk 1]
```diff
@@ -91,26 +98,26 @@ def recognize_face(self, frame):
                 color = (0,35,200)
                 is_authorized = "False"
             # Draw bounding box around the detected face
-            l= 20
-            t= 5
+            offset= 20
+            thickness= 5
```
Comment (sanskaromar) [inline app/security_system.py:102]: Nice.

**Review by sanskaromar — APPROVED** [review]
Good Work 🚀

File: app/security_system.py (hunk 1 above)
Comment (Aqua-16) [reply app/security_system.py:102]: Thanks 😄

**Pull request merged**